deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -57,7 +57,31 @@ spec:
       {{- end }}
       {{- if $options.hasConfigMap }}
       shareProcessNamespace: true
+      {{- end }}
       initContainers:
+      - image: ubuntu
+        name: gpu-feature-discovery-imex-init
+        command:
+        - "/bin/bash"
+        - "-c"
+        - |
+          if [[ ! -d /driver-root/etc/nvidia-imex ]]; then
+            echo "No IMEX Nodes config path detected; Skipping"
+            exit 0
+          fi
+
+          mkdir -p /config/etc/nvidia-imex
+          mount --bind /driver-root/etc/nvidia-imex /config/etc/nvidia-imex
+        volumeMounts:
+          - name: config
+            mountPath: /config
+            mountPropagation: Bidirectional
+          - name: driver-root
+            mountPath: /driver-root
+            readOnly: true
+        securityContext:
+          privileged: true
```

Comment:
If you make this the second init container does it remove the need for two separate ` {{- if $options.hasConfigMap }}` checks?

Reply:
I don't think so due to the `shareProcessNamespace: true` that is required for the config sidecare container.

Reply:
I see -- yeah, it's because you need to unconditionally create the initContainers section between setting shareProcessNamespace: true and defining the config init container.